misc/acrn-config/launch_config/com.py

```diff
@@ -447,8 +462,6 @@ def dm_arg_set(names, sel, dm, vmid, config):
     # hard rt os
     if uos_type == "PREEMPT-RT LINUX":
         print("acrn-dm -A -m $mem_size -s 0:0,hostbridge -U {} \\".format(scenario_uuid[vmid + sos_vmid]), file=config)
-        #print("   -k /usr/lib/kernel/default-iot-lts2018-preempt-rt \\", file=config)
-        print("   -k /usr/lib/kernel/default-iot-lts2018-preempt-rt \\", file=config)
```

Comment:
we totally disable the option to boot the kernel directly, right?

Reply:
Yes, as webUI has not such configuration, so currently, we disable the '-k' option.
We also plan to enable it on next release while webUI supports such setting.